### DISCUSSION TITLE
AEON Chip

### DISCUSSION THREAD

**Discussion by 1hedu** [uxmal/reko#1245] [General]
It would appear that I am reversing an AEON type chip. I am getting a lot of Nyi instructions. Any tips for AEON? I assume since the instruction set has been mostly reversed, that a listing of it exists somewhere. Where can I see that? Also since it is known, does that mean that binwalk and other tools could be updated with some AEON recognition? 

Completely new to Reko, very happy it supports AEON.

Many thanks

**Comment by throwaway96** [comment]
The most complete list of instructions is in `AeonDisassembler.cs`. Note that most of the instructions are unconfirmed guesses.

**Reply by 1hedu** [reply]
Thx

**Reply by throwaway96** [reply]
You can join us on [Discord](https://discord.gg/9PMWVxEjqr) if you'd like to talk about MStar/AEON stuff.
